include/mapbox/geometry/wagyu/util.hpp

```diff
@@ -30,18 +31,27 @@ double area(mapbox::geometry::linear_ring<T> const& poly) {
     return -a * 0.5;
 }
 
-inline bool value_is_zero(double val) {
-    return std::fabs(val) < (5.0 * std::numeric_limits<double>::epsilon());
+inline bool values_are_equal(double x, double y) {
+    return util::FloatingPoint<double>(x).AlmostEquals(util::FloatingPoint<double>(y));
 }
 
-inline bool values_are_equal(double x, double y) {
-    return value_is_zero(x - y);
+inline bool value_is_zero(double val) {
+    return values_are_equal(val, static_cast<double>(0.0));
 }
 
 inline bool greater_than_or_equal(double x, double y) {
     return x > y || values_are_equal(x, y);
 }
 
```

Comment:
this is a little pedantic, but might be nice to also add a `less_than_or_equal` fxn to complete the set. maybe:

```suggestion

inline bool less_than_or_equal(double x, double y) {
    return x < y || values_are_equal(x, y);
}

```

Reply:
I am not going to add it if it is not used currently.